easybuild/easyconfigs/b/biogeme/biogeme-3.1.2-foss-2021a-Python-3.9.5.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'biogeme'
+version = '3.1.2'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
@benczaja We don't use the `Python` `versionsuffix` anymore in this toolchain, so please remove from the `easyconfig` and from the filename.

Reply:
@verdurin `versionsuffix` removed from both config and filename. Thanks!